### PR TITLE
Update chineseSimplified.xml

### PR DESCRIPTION
**Pull request by popcorner**
merge all the changes in english.xml

### CODE REVIEW

**Comment by donho** [comment]
@popcorner 
Thank you for your update. 
Could you check the diff from v7.8.9 to the newest `english.xml` to make sure there's nothing's missing?
http://download.notepad-plus-plus.org/repository/MISC/diff/english.diff.htm

**Comment by popcorner** [comment]
@donho new changes are added. Now it's up to date.

File: PowerEditor/installer/nativeLang/chineseSimplified.xml [hunk 1]
```diff
@@ -970,17 +1043,17 @@
...
-                <Item id="2025" name="八进制"/>
-                <Item id="2026" name="十六进制"/>
-                <Item id="2027" name="二进制"/>
+                <Item id="2023" name="插入文本(&amp;T)"/>
+                <Item id="2033" name="插入数字(&amp;N)"/>
+                <Item id="2030" name="初始值(&amp;I) :"/>
+                <Item id="2031" name="增量(&amp;Y) :"/>
+                <Item id="2035" name="开头补零(&amp;Z)"/>
+                <Item id="2036" name="重复次数(&amp;R) :"/>
+                <Item id="2032" name="格式(&amp;F)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/chineseSimplified.xml:1052]: Sorry I just fixed an issue regarding it: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/55d671719c8f00f02ed32b15575bc2e6587a5e30
`&amp;F` should be removed.

**Review by donho — CHANGES_REQUESTED** [review]
Please remove `(&amp;F)` for Format.

File: PowerEditor/installer/nativeLang/chineseSimplified.xml (hunk 1 above)
Comment (popcorner) [reply PowerEditor/installer/nativeLang/chineseSimplified.xml]: @donho new changes in 55d6717 are added.

**Review by donho — APPROVED** [review]

**Pull request closed without merging**